airflow/config_templates/default_airflow.cfg

```
@@ -692,6 +692,9 @@ password =
 # The default owner assigned to each new operator, unless
 # provided explicitly or passed via ``default_args``
 default_owner = airflow
+
+# The default value of attribute "deferrable" in BaseDeferrableOperator.
+default_deferrable = false
```

```suggestion
default_deferrable = False
```

The boolean values in this file seem to be in Title case

---

Sure. Thanks for your suggestion! I also changed other `false` in this file to `False`